GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -224,7 +224,7 @@ protected virtual void OnDetached()
             /// <summary>
             /// Requests to refresh the data tree and to apply filtering, if necessary.
             /// </summary>
-            protected internal virtual void Refresh()
+            protected internal virtual void Refresh(Func<RefsFilter, IReadOnlyList<IGitRef>> refs)
```

Comment:
❔ Shall the new argument be used by overridden functions only?

Reply:
No, it will be used on line 235, missed that when backporting
(and will change to getRefs)